schema-registry/schema-registry-docker.sh

```diff
@@ -1,21 +1,32 @@
 #!/bin/bash
 
-: ${SCHEMA_REGISTRY_PORT:=8081}
-: ${SCHEMA_REGISTRY_KAFKASTORE_TOPIC:=_schemas}
-: ${SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL:=$ZOOKEEPER_PORT_2181_TCP_ADDR:$ZOOKEEPER_PORT_2181_TCP_PORT}
-: ${SCHEMA_REGISTRY_DEBUG:=false}
+sr_cfg_file="/etc/schema-registry/schema-registry.properties"
 
-export SCHEMA_REGISTRY_PORT
-export SCHEMA_REGISTRY_KAFKASTORE_TOPIC
-export SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL
-export SCHEMA_REGISTRY_DEBUG
+: ${SR_PORT:=8081}
```

Comment:
You can call the link alias anything you want so it would be `--link schema-registry:sr` where `schema-registry` is what you called the Schema Registry container, `docker run --name schema-registry`, and `sr` is my alias so the variables become `SR_`. This change would require a README update which I thought was in one of my PRs (I know, it should have been in this). We just prefer short names since it's less to type when coding but it can stay as you prefer for wider clarity :-).